streams/src/main/java/org/apache/kafka/streams/state/internals/MeteredKeyValueStore.java

```diff
@@ -229,6 +230,15 @@ public V delete(final K key) {
         }
     }
 
+    @Override
+    public <PS extends Serializer<P>, P> KeyValueIterator<K, V> prefixScan(final P prefix, final PS prefixKeySerializer) {
+
+        return new MeteredKeyValueIterator(
+                wrapped().prefixScan(prefix, prefixKeySerializer),
+                rangeSensor
+        );
```

Comment:
nit: Sorry, I overlooked this last time. We actually use 4 space and not 8.
```suggestion
        return new MeteredKeyValueIterator(
            wrapped().prefixScan(prefix, prefixKeySerializer),
            rangeSensor
        );
```